src/modules/comments.js

```diff
@@ -1,33 +1,44 @@
+import commentsCounter from './commentsCounter'
 
-const showComments = comments => {
-    const content = document.querySelector('.comment-section')
-	comments.forEach((comment, index) => {
-		if (index < 20) {
-			content.innerHTML += `
+const showPopups = async (data, updateComment) => {
+	const comments = await updateComment(data.name)
+	console.log(comments)
```

Comment:
- [ ] For best coding practice, i will strongly recommend you avoid leaving `console.logs()` in your code base as it has the potential of reducing the overall performance of your website 👍